src/progress_bar.rs

```diff
@@ -478,7 +480,7 @@ impl ProgressBar {
 
     /// Returns the current rate of progress
     pub fn per_sec(&self) -> u64 {
-        self.state.lock().unwrap().per_sec()
+        self.state.lock().unwrap().per_sec() as u64
```

Comment:
How do you feel about just making this also return `f64`?

Reply:
`f64` is probably more consistent, so I think it'd be better.